diff --git a/Application/Services/TaskService.cs b/Application/Services/TaskService.cs
--- a/Application/Services/TaskService.cs
+++ b/Application/Services/TaskService.cs
@@ -38,7 +38,7 @@
     public async Task<ResponseModel<TaskDto>> CreateTaskAsync(TaskDto dto)
     {
         var entity = _mapper.Map<TaskEntity>(dto);
         await _context.Tasks.AddAsync(entity);
         await _context.SaveChangesAsync();
-        return new ResponseModel<TaskDto>() { Success = true, Message = "Created", Data = _mapper.Map<TaskDto>(entity) };
+        return new ResponseModel<TaskDto>() { Success = true, Message = "Task created successfully", Data = _mapper.Map<TaskDto>(entity) };
     }

